src/details/ArborX_DetailsTreeConstruction.hpp

```diff
@@ -57,18 +57,34 @@ inline void projectOntoSpaceFillingCurve(ExecutionSpace const &space,
       });
 }
 
-template <typename ExecutionSpace, typename Values, typename Nodes>
-inline void initializeSingleLeafNode(ExecutionSpace const &space,
-                                     Values const &values,
-                                     Nodes const &leaf_nodes)
+template <typename ExecutionSpace, typename Values, typename IndexableGetter,
+          typename Nodes, typename BoundingVolume>
+inline void
+initializeSingleLeafTree(ExecutionSpace const &space, Values const &values,
+                         IndexableGetter const &indexable_getter,
+                         Nodes const &leaf_nodes, BoundingVolume &bounds)
 {
   ARBORX_ASSERT(leaf_nodes.extent(0) == 1);
   ARBORX_ASSERT(values.size() == 1);
 
+  // Skip initialization so that we don't execute a kernel launch
+  Kokkos::View<BoundingVolume, typename Nodes::memory_space> bounding_volume(
+      Kokkos::view_alloc(space, Kokkos::WithoutInitializing,
+                         "ArborX::BVH::getSingleLeafBounds::bounding_volume"));
```

Comment:
```suggestion
  using MemorySpace = typename Nodes::memory_space;
  auto bounding_volume = Kokkos::create_mirror_view(
      Kokkos::view_alloc(space, Kokkos::WithoutInitializing,
                         "ArborX::BVH::getSingleLeafBounds::bounding_volume",
                         MemorySpace()),
      Kokkos::View<BoundingVolume, Kokkos::HostSpace, Kokkos::MemoryUnmanaged>(
          &bounds));
```

Reply:
OK, this does not work because of the labels. `view_alloc` in `create_mirror_view` does not allow a label. Which results in `bounding_volume` having a label `_mirror`, which fails out labels tests.

Reply:
This is a bug on the Kokkos side.  I am tempted to just filter out `"_mirror"`.

Reply:
Is the bug that it does not allow a label?

Reply:
Alternatively, can just do what I did before.